fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/VehicleUnit.java

```diff
@@ -169,7 +169,7 @@ public void changeLane(VehicleLaneChange.VehicleLaneChangeMode vehicleLaneChange
 
 
     @Override
-    public void slowDown(float speed, int interval) {
+    public void slowDown(float speed, long interval) {
         VehicleSlowDown vehicleSlowDown = new VehicleSlowDown(
```

Comment:
Could you change the field timeInterval in VehicleSlowDown to long/nanoseconds too? For uniformity between the classes related to vehicle control.

Reply:
Is implemented